ruby/vampires.rb

```diff
@@ -0,0 +1,50 @@
+puts "How many employees will be processed?"
+x = gets.to_i
+x.times do 
+
+	puts "What is your name?"
+	user_name = gets.chomp
+
+	puts "How old are you?"
+	user_age = gets.to_i
+
+	puts "what year were you born?"
+	user_birthyear = gets.to_i
+	user_trueage = user_age + user_birthyear
```

Comment:
By adding the age + birth year, you should get the current year. That logic makes sense! The only issue is that when you are checking this condition in your if/else you have to code the current year (2016). This would mean that if your code lasted for more then a year it would have to be updated manually. 

How could you structure this so that it does not rely on 'magic numbers'?